vendordeps/REVLib.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
This should probably be part of the BaseCode.

Reply:
I forgot to delete it as its not used. But It should be in base code agreed.